raiden_contracts/data_0.37.0/source/raiden/SecretRegistry.sol

```diff
@@ -41,7 +41,7 @@ contract SecretRegistry {
     /// @notice Get the stored block number at which the secret was revealed
     /// @param secrethash The hash of the registered secret `keccak256(secret)`
     /// @return The block number at which the secret was revealed
-    function getSecretRevealBlockHeight(bytes32 secrethash) public view returns (uint256) {
+    function getSecretRevealBlockTime(bytes32 secrethash) public view returns (uint256) {
```

Comment:
All files in the subfolders `data_X.X.X` should not be changed, as they're snapshots of the contracts at the given version.

Reply:
✅